### ISSUE TITLE
Android: localization

### ISSUE THREAD

**Issue by adrenalin8231** [google/project-gameface#54]
tl;dr: Do you have any commits or future plans to refactor the Android codes for proper localization support? If not, will you accept PR?

---

Greetings from South Korea,

My friends with limited motor control showed interest in this project and shared it to me. I built the app, installed it on my phone, and demonstrated with them. The app itself was great.
Now they want Korean language support for the app before installing it on their phones. They will have to find best gesture/cursor settings to try GameFace, and they want to do it on their own, w/o help of English-capable person like me.

Cutrently GameFace Android has hard-coded English strings in Java code, such as:
https://github.com/google/project-gameface/blob/daceab7bc30b617cd56dd2e528967bbb2d1fa209/Android/app/src/main/java/com/google/projectgameface/BlendshapeEventTriggerConfig.java#L59-L60
so I couldn't do full localizing before refactoring. I managed to do them on my fork, and it seems to work on my device.
The thing is, since I am a JS/TS web programmer and have very limited experience on Java/Kotlin, I don't have guts to upload PR. (I made https://github.com/google/project-gameface/pull/53 but this is by accident, sorry)

Do you have any commits coming, or at least future plans to support full localization? If not, can I reopen my PR and get help to get them merged?
